### PR TITLE
tests: fix tests exit failure

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang @disksing PTAL

### CODE REVIEW

File: tests/raftstore/util.rs [hunk 1]
```diff
@@ -69,6 +69,10 @@ pub fn new_store_cfg() -> Config {
         raft_heartbeat_ticks: 2,
         raft_election_timeout_ticks: 20,
         raft_log_gc_tick_interval: 100,
+        // because our tests may quit very soon, so we take a large
+        // value to avoid pd ask failure.
+        // TODO: should we ignore all mock pd failure?
+        replica_check_tick_interval: 60 * 1000,
```
Comment (siddontang) [inline tests/raftstore/util.rs]: does auto change test use little interval?

Comment (BusyJay) [inline tests/raftstore/util.rs]: Yes. This configuration is 4 sec by default. Auto conf change use 200 millis.

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**